documentation/src/mkdocs.yml

```diff
@@ -13,26 +13,60 @@ site_description: 'Documentation to help you get familiar with Pravega'
 repo_url: 'https://github.com/pravega/pravega'
 edit_uri: 'tree/master/documentation/src/docs'
 
-theme: 'material'
+# theme: 'readthedocs'
+theme:
+  name: material
+  custom_dir: custom_theme
+
+  # 404 page
+  #static_templates:
+  #  - 404.html
+
+  # Necessary for search to work properly
+  #include_search_page: false
+  #search_index_only: true
+
+  # Default values, taken from mkdocs_theme.yml
+  #language: en
+  #font:
+  #  text: Roboto
+  #  code: Roboto Mono
+  favicon: assets/favicon.ico
+  #icon:
+  #  logo: logo
 
 
 # Extensions
 markdown_extensions:
   - admonition
-  - codehilite(guess_lang=false)
+  - codehilite:
+      guess_lang: false
   - footnotes
   - meta
-  - toc(permalink=true)
-  - pymdownx.betterem(smart_enable=all)
+  - toc:
+      permalink: true
+  - pymdownx.betterem:
+      smart_enable: all
   - pymdownx.caret
   - pymdownx.inlinehilite
   - pymdownx.magiclink
   - pymdownx.smartsymbols
   - pymdownx.superfences
+  - pymdownx.highlight
 
-pages:
+nav:
   - Overview: 'index.md'
-  - Getting Started: 'getting-started.md'
+  - Quick Start Guide: 'getting-started/quick-start.md'
+  - Getting Started Guide: 'getting-started.md'
+  - Admin Guide:
+    - 'Cluster Overview': 'pravega-operator/doc/cluster-overview.md'
+    - 'Deploy Pravega': 'pravega-operator/charts/pravega/README.md'
+    - 'Upgrade Pravega': 'pravega-operator/doc/upgrade-cluster.md'
+    - 'Configure Pravega': 'pravega-operator/doc/configuration.md'
+  - Dev Guide:
```

Comment:
I also feel that the content here is overlapping with section `Developing Pravega Applications`. We may want to merge these into a single section and be sure that we are not duplicating content.